tests/src/az_ulib_test_my_interface.c

```diff
@@ -227,10 +227,9 @@ static az_result my_command_span_wrapper(az_span model_in_span, az_span* model_o
 }
 
 /*
- * Publish MY_INTERFACE_1_V123
+ * Publish MY_INTERFACE_A_1_1_123
  */
-static const az_ulib_capability_descriptor
-    MY_INTERFACE_1_V123_CAPABILITIES[MY_INTERFACE_1_123_CAPABILITY_SIZE]
+static const az_ulib_capability_descriptor MY_INTERFACE_A_1_1_123_CAPABILITIES[]
```

Comment:
>A_1_1_123 [](http://example.com/codeflow?start=56&length=9)

what do these represent?

Reply:
This is not a standard, just a way to make easy for the test, it represents:
```
    MY_PACKAGE_A_NAME,
    MY_PACKAGE_1_VERSION,
    MY_INTERFACE_1_NAME,
    MY_INTERFACE_123_VERSION,
```